### PR TITLE
Add v1.0.0 release notes to changelog

### PR DESCRIPTION
**Pull request by MaciejBaj**
## Summary
(Summarize the changes that this pull request creates)-

## Checklist
- [ ] insert any notable changes, in order of commit

## Please check that your PR completes the requirements as follows
- [ ] Tests have been added for bug fixes/features
- [ ] Acceptance criteria from the issue has been completed

### CODE REVIEW

**Review by AwesomeIbex — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## `v1.0.0` 2022-10-28
+#### Shell Runtime of t3rn
+
```
Comment (chiefbiiko) [inline CHANGELOG.md:5]:
```suggestion
* no custom features
* open collator pool
* upgradable runtime
```

File: .github/workflows/circuit-t3rn-collator-release.yml [hunk 2]
```diff
@@ -86,9 +86,9 @@ jobs:
         with:
           context: .
           push: true
-          tags: t3rn/t3rn-collator:${{ env.PUSHED_TAG }}
+          tags: ghcr.io/t3rn/t3rn-collator:${{ env.PUSHED_TAG }}
           platforms: linux/amd64
-          file: collator.Dockerfile
+          file: collator.t3rn.Dockerfile
```
Comment (chiefbiiko) [inline .github/workflows/circuit-t3rn-collator-release.yml:91]: `t3rn.collator.Dockerfile` ?

File: collator.t3rn.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ENV PARACHAIN_NAME=t3rn
+
+RUN useradd -m -u 1000 -U -s /bin/sh -d /node runner && \
+    mkdir /node/data && \
+    mkdir /node/keystore && \
+    mkdir /node/specs
+
+COPY --chown=runner target/release/${PARACHAIN_NAME}-collator /usr/local/bin/${PARACHAIN_NAME}-collator
+COPY --chown=runner specs/$RELAYCHAIN_NAME.raw.json /node/specs/$RELAYCHAIN_NAME.raw.json
+COPY --chown=runner specs/$PARACHAIN_NAME.raw.json /node/specs/$PARACHAIN_NAME.raw.json
```
Comment (chiefbiiko) [inline collator.t3rn.Dockerfile:14]: We should also regen and check-in the t3rn specs under `./specs` ontop of the pure shell runtime tag.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## `v1.0.0` 2022-10-28
```
Comment (chiefbiiko) [inline CHANGELOG.md:3]:
```suggestion
## `v0.1.0` 2022-10-28
```

Shouldn't a `v1.0.0` by convention incl at least a minimal feature set? Would reserve the `v1.0.0` for our MVP.

**Review by chiefbiiko — CHANGES_REQUESTED** [review]

File: collator.t3rn.Dockerfile (hunk 3 above)
Comment (MaciejBaj) [reply collator.t3rn.Dockerfile:14]: done

**Pull request merged**